frame/contracts/src/lib.rs

```diff
@@ -255,10 +255,29 @@ pub mod pallet {
 		type CallStack: smallvec::Array<Item = Frame<Self>>;
 
 		/// The maximum number of tries that can be queued for deletion.
```

Comment:
Slightly ambiguous as "tries" could possibly be interpreted as "attempts" on first reading. Maybe fully qualify e.g. "contract storage tries"

Reply:
Good point. Changed the wording.